Jenkinsfile.aws-test

```diff
@@ -32,6 +35,7 @@ node(NODE) {
             sshUserPrivateKey(credentialsId: params.ARTIFACT_SSH_CREDS_ID, keyFileVariable: 'KEY_FILE'),
         ]) {
             utils.rsync_file_in_dest(ARTIFACT_SERVER, KEY_FILE, "${images}/cloud/latest/meta.json", "${WORKSPACE}/meta.json")
+            ostree_commit = utils.sh_capture("jq -r '.[\"ostree-commit\"]' ${WORKSPACE}/meta.json")
```

Comment:
This isn't new (you're building on code right below), but probably we should just use `readJSON` rather than calling out to `jq` - Groovy is a **real** programming language.

Reply:
Okay, I'll include another commit to use that in this area.